components/script/webdriver_handlers.rs

```diff
@@ -270,6 +272,49 @@ pub fn handle_get_browsing_context_id(
     reply.send(result).unwrap()
 }
 
+fn get_element_in_view_center_point(element: &Element) -> Option<Point2D<i64>> {
+    match element.GetClientRects().iter().next() {
```

Comment:
Why are we getting all the rects and only using the first one?

Reply:
That's the behavior from the spec: https://w3c.github.io/webdriver/#dfn-center-point.

Reply:
Let's add a spec link and some comments with step numbers, in that case :)